flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -42,9 +42,12 @@ jobmanager.rpc.port: 6123
 jobmanager.heap.size: 1024m
 
 
-# The heap size for the TaskManager JVM
+# The total process memory size for the TaskManager.
+#
+# Note this accounts for all memory usages of a TaskManager process, including JVM metaspace and other overheads.
+# To exclude JVM metaspace and other overheads, please use total flink memory size (taskmanager.memory.flink.size) instead.
 
-taskmanager.memory.process.size: 1024m
+taskmanager.memory.process.size: 1568m
```

Comment:
```suggestion

# To exclude JVM metaspace and overhead, please, use total Flink memory size instead of 'taskmanager.memory.process.size'.
# It is not recommended to set both 'taskmanager.memory.process.size' and Flink memory.
# This option is similar to the deprecated option 'taskmanager.heap.size' for standalone environment:
#
# taskmanager.memory.flink.size: 1280
```